src/content/landing.ts

```diff
@@ -155,9 +155,8 @@ const typedLanding: CustomPageContent = {
             color: '#23151e',
             image: bannerAsset,
             commands: [
-                'git clone git@github.com:gravity-ui/uikit-example-cra.git my-project && cd my-project',
-                'npm i',
-                'npm run start',
+                'npx create-react-app ./my-project --template gravity-ui && cd my-project',
+                'npm run dev',
```

Comment:
1) After installing and "npm run dev" I've seen a blank page.
2) Why should we use `create-react-app` if we use `app-builder` in template and don't use `react-scripts`? It is not actually a CRA application.

Reply:
1. Hmm, have you opened the app on `http://localhost:8000/`?
2. Good point. I've removed cra call, and will change repo name later, is it ok?

Reply:
it works on port 8000, but:
<img width="1087" alt="Screenshot 2023-09-24 at 14 28 58" src="https://github.com/gravity-ui/landing/assets/29424746/747d09b5-c7f5-4f5f-9e2b-ef88b2a72520">